db/schema.rb

```diff
@@ -630,7 +630,6 @@
   add_foreign_key "participant_declarations", "lead_providers"
   add_foreign_key "participant_profiles", "cohorts"
   add_foreign_key "participant_profiles", "core_induction_programmes"
-  add_foreign_key "participant_profiles", "participant_profiles", column: "mentor_profile_id"
```

Comment:
I don't think schema should change in this PR - we don't have any migrations here.

Reply:
It is likely this should not go into schema in previous PR... This FK won't work until we have all the recrods moved over to the new table and will be added in next PR...

Reply:
ok, so is this schema the result of running migrations from scratch?

Reply:
Yes :)